markdown/events/2022/bytesize-nanoseq.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Bytesize: nf-core/nanoseq'
+subtitle: Yuk Kei Wan - National University of Singapore
```

Comment:
Do you mind putting Genome Institute of Singapore and National University of Singapore as my affiliations, please? Thanks!

Reply:
of course, no problem! :)